src/coreclr/jit/emitxarch.cpp

```diff
@@ -11568,12 +11595,12 @@ BYTE* emitter::emitOutputCV(BYTE* dst, instrDesc* id, code_t code, CnsVal* addc)
             code &= 0x0000FFFF;
         }
 
-        if ((ins == INS_movsx || ins == INS_movzx || ins == INS_cmpxchg || ins == INS_xchg || ins == INS_xadd ||
-             insIsCMOV(ins)) &&
-            size != EA_1BYTE)
+        // Anthony: Per L10986, why is this insIsCMOV (which does not have a w bit)
+        // and above is !insIsCMOV
```

Comment:
I can't say exactly why these checks are there, but since we currently do not generate cmov I would be fine with removing the checks and leave it up to future work to make sure the emitter supports them if we decide to start generating it again.

Reply:
Ok, I have gone ahead and removed the comments and the redundant cmov checks.